### PR TITLE
Fix the events-based objects editor that were never showing up in dev mode.

### PR DESCRIPTION
**Pull request by D8H**

### CODE REVIEW

File: newIDE/app/src/UI/EditorMosaic/index.js [hunk 1]
```diff
@@ -39,10 +39,13 @@ export const mosaicContainsNode = (
...
     (mosaic === node ||
-      (!!mosaic.first &&
-        mosaicContainsNode(((mosaic.first: ?any): ?EditorMosaicNode), node) &&
-        !!mosaic.second &&
-        mosaicContainsNode(((mosaic.second: ?any): ?EditorMosaicNode), node)))
+      ((!!mosaic.first &&
+        mosaicContainsNode(((mosaic.first: ?any): ?EditorMosaicNode), node)) ||
+        (!!mosaic.second &&
+          mosaicContainsNode(
+            ((mosaic.second: ?any): ?EditorMosaicNode),
```
Comment (4ian) [inline newIDE/app/src/UI/EditorMosaic/index.js:46]: // $FlowFixMe might have be maybe simpler than the double cast?

File: newIDE/app/src/EventsFunctionsExtensionEditor/index.js [hunk 2]
```diff
@@ -1410,9 +1410,11 @@ export default class EventsFunctionsExtensionEditor extends React.Component<
                       )
                     }
                     initialNodes={
-                      isDev !==
+                      isDev ===
```
Comment (4ian) [inline newIDE/app/src/EventsFunctionsExtensionEditor/index.js:1413]: I'm not sure to follow why isDev is compared to mosaicContainsNode? Any chance you can rework this ternary?

**Review by 4ian — APPROVED** [review]
Approving to unblock but the logic is hard to parse/read, see if you can rework it?

File: newIDE/app/src/EventsFunctionsExtensionEditor/index.js (hunk 2 above)
Comment (D8H) [reply newIDE/app/src/EventsFunctionsExtensionEditor/index.js:1413]: In development mode, the object list must be shown: the mosaic must be reset to default if the save settings don't contain it.
This will be removed as soon as we release the events-based objects editor.

**Review by 4ian — APPROVED** [review]
